### PR TITLE
updated code

### PR DESCRIPTION
**Pull request by Vaibhavkt2**
Review This Code And Let Me Know if this correct or Not

### CODE REVIEW

**Comment by sanketp9972** [comment]
okay faulty work

File: index.html [hunk 1]
```diff
@@ -5,7 +5,9 @@
 </head>
 <body>
     <h1>Hii EveryOne</h1>
-    <h2>Love You JaggaDaku</h2>
-    <h2>Love You JaggaDaku</h2>
+    <h2>Love You Jagurati</h2>
+    <h3>Sanket Chi Sankita</h3>
```
Comment (SUYOG279) [inline index.html:9]: Sanketa Chai Vali Bolo Juba Kesari

Comment (Vaibhavkt2) [reply index.html:9]: Goli

Comment (SUYOG279) [reply index.html:9]: Shubhu

**Pull request merged**